lib/tasks/curation_concerns.rake

```diff
@@ -10,18 +10,9 @@ namespace :curation_concerns do
   namespace :migrate do
     desc "Migrate collections"
     task collections: :index do
-      Collection.all.each do |collection|
-        collection.members.each do |member|
-          member.member_of_collections << collection
-          member.save
-        end
-        collection.members = []
-        collection.save
-      end
+      CurationConcerns::DataMigration::CollectionsMigration.run
     end
   end
-end
-namespace :curation_concerns do
```

Comment:
Do we need to leave this in, or won't it change the namespace of the following rake task?